k9mail/src/main/java/com/fsck/k9/fragment/MessageListFragment.java

```diff
@@ -945,56 +942,35 @@ private void restartLoader() {
         }
     }
 
-    private void initializePullToRefresh(LayoutInflater inflater, View layout) {
-        mPullToRefreshView = (PullToRefreshListView) layout.findViewById(R.id.message_list);
-
-        @SuppressLint("InflateParams")
-        View loadingView = inflater.inflate(R.layout.message_list_loading, null);
-        mPullToRefreshView.setEmptyView(loadingView);
+    private void initializePullToRefresh(View layout) {
+        mSwipeRefreshLayout = (SwipeRefreshLayout) layout.findViewById(R.id.swiperefresh);
+        mListView = (ListView) layout.findViewById(R.id.message_list);
 
         if (isRemoteSearchAllowed()) {
-            // "Pull to search server"
-            mPullToRefreshView.setOnRefreshListener(
-                    new PullToRefreshBase.OnRefreshListener<ListView>() {
+            mSwipeRefreshLayout.setOnRefreshListener(
+                    new SwipeRefreshLayout.OnRefreshListener() {
                         @Override
-                        public void onRefresh(PullToRefreshBase<ListView> refreshView) {
-                            mPullToRefreshView.onRefreshComplete();
+                        public void onRefresh() {
                             onRemoteSearchRequested();
```

Comment:
previously, this immediately reset the "refreshing" state. does this change behavior?